src/libraries/System.Net.Quic/src/System.Net.Quic.csproj

```
@@ -137,6 +137,9 @@
   </ItemGroup>
 
   <ItemGroup>
+    <ProjectReference Include="$(LibrariesProjectRoot)System.Net.Security\src\System.Net.Security.csproj" >
+        <SkipUseReferenceAssembly>true</SkipUseReferenceAssembly>
+    </ProjectReference>
```

nit: less verbose syntax

```suggestion
    <ProjectReference Include="$(LibrariesProjectRoot)System.Net.Security\src\System.Net.Security.csproj"
                      SkipUseReferenceAssembly="true" />
```